src/main/java/mate/jdbc/dao/ManufacturerDaoImpl.java

```diff
@@ -0,0 +1,131 @@
+            if (resultSet.next()) {
+                return Optional.of(createManufacturerObject(resultSet));
+            }
+            return Optional.empty();
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't create request for find manufacturer", e);
+        }
+    }
+
+    @Override
+    public List<Manufacturer> getAll() {
+        List<Manufacturer> manufacturers = new ArrayList<>();
+        String getAllManufacturersRequest =
+                "SELECT * FROM manufacturers WHERE is_deleted = FALSE";
+        try (Connection connection = ConnectionUtilImpl.getConnection();
+                Statement statement = connection.createStatement()) {
+            ResultSet resultSet = statement.executeQuery(getAllManufacturersRequest);
+            while (resultSet.next()) {
+                Manufacturer newManufacturer = createManufacturerObject(resultSet);
+                manufacturers.add(newManufacturer);
+            }
+            return manufacturers;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't create request for return all DB", e);
+        }
+    }
+
+    @Override
+    public Manufacturer update(Manufacturer manufacturer) {
+        String updateManufacturerRequest =
+                "UPDATE manufacturers SET name = ?, country = ? WHERE id = ?";
+        try (Connection connection = ConnectionUtilImpl.getConnection();
+                PreparedStatement updateStatement =
+                        connection.prepareStatement(updateManufacturerRequest)) {
+            updateStatement.setString(1, manufacturer.getName());
+            updateStatement.setString(2, manufacturer.getCountry());
+            updateStatement.setLong(3, manufacturer.getId());
+            updateStatement.executeUpdate();
+            return manufacturer;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't create update request for input data. "
+                    + "Manufacturer: id = "
+                    + manufacturer.getId()
+                    + ", name = "
+                    + manufacturer.getName()
+                    + ", country = "
+                    + manufacturer.getCountry(), e);
+        }
+    }
+
+    @Override
+    public boolean delete(Long id) {
+        String deleteManufacturerByIdRequest
+                = "UPDATE manufacturers SET is_deleted = TRUE WHERE id = ?";
+        try (Connection connection = ConnectionUtilImpl.getConnection();
+                PreparedStatement deleteManufacturerStatement
+                        = connection.prepareStatement(deleteManufacturerByIdRequest)) {
+            deleteManufacturerStatement.setLong(1, id);
+            return deleteManufacturerStatement.executeUpdate() > 0;
+        } catch (SQLException e) {
+            throw new DataProcessingException(
+                    "Can't create delete request for input params. Manufacturer id = " + id, e);
```

Comment:
```suggestion
                    "Can't execute delete request for input params. Manufacturer id = " + id, e);
```
let's better change `create request` to `execute request` for all your exception messages
we can create a request query, but there might be some error in progress of execution

Reply:
done